.github/workflows/build.yml

```diff
@@ -63,6 +63,8 @@ jobs:
           export CUDA_HOME=/usr/local/cuda-${CUDA_SHORT}
           export LD_LIBRARY_PATH=${CUDA_HOME}/lib64:${CUDA_HOME}/include:${LD_LIBRARY_PATH}
           export PATH=${CUDA_HOME}/bin:${PATH}
+      - name: Install wheel
```

Comment:
Should we also put it as a dependency of mim?

Reply:
It is probably a problem of docker that `wheel` is not default installed

Reply:
https://github.com/zhouzaida/mim/runs/3146423873?check_suite_focus=true#step:6:50

if the wheel is not installed, the page-home will be `'/opt/hostedtoolcache/Python/3.7.11/x64/lib/python3.7/site-packages'` rather than `https://github.com/open-mmlab/mmclassification`. The latter string is what we want to obtain

Reply:
Then should leave a comment here because it seems that the env has such a package.

Reply:
got it